test/unit/spec/common/cellComponents/tabs/jobStatusTable-Spec.js

```diff
@@ -645,45 +691,27 @@ define([
 
             // make sure that the row contents are correct
             JobsData.allJobs.forEach((job) => {
-                describe(`${job.job_id} row content`, () => {
-                    describe('in a batch table', () => {
-                        beforeEach(async function () {
-                            container = document.createElement('div');
-                            this.job = TestUtil.JSONcopy(job);
-                            const batchParent = {
-                                batch_id: this.job.batch_id,
-                                job_id: this.job.batch_id,
-                                batch_job: true,
-                                child_jobs: [this.job.job_id],
-                            };
-                            this.jobManager = new JobManager({
-                                model: makeModel([this.job, batchParent]),
-                                bus: Runtime.make().bus(),
-                            });
-                            this.jobStatusTableInstance = await createStartedInstance(container, {
-                                jobManager: this.jobManager,
-                            });
-                            this.row = container.querySelector('tbody tr');
+                describe(`row content, ${job.job_id}`, () => {
```

Comment:
no significant content change, just decrease in indent due to removing the dupe test